frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -44,6 +44,11 @@ import { AcmForm, AcmLabelsInput, AcmModal, AcmSelect, AcmSubmit, Provider } fro
 import { CredentialsForm } from '../../Credentials/CredentialsForm'
 import schema from './schema.json'
 
+const DEFAULT_DESTROY_TIMEOUT = 5
+const DEFAULT_INSTALL_TIMEOUT = 5
+const DEFAULT_SCALE_TIMEOUT = 5
+const DEFAULT_UPGRADE_TIMEOUT = 480
```

Comment:
@jeswanke You were looking at a hardcoded timeout for the `release-2.6` branch. The timeout for MCE 2.3.0 will be 120:
https://github.com/stolostron/cluster-curator-controller/blob/backplane-2.3/pkg/jobs/hive/hive.go#L435

Also, you did not update the tooltip message to match. Please use a `{{timeout}}` variable in the messages so if these ever change we can change them in 1 spot only.